### ISSUE TITLE
The tonal is not a phrase !

### ISSUE THREAD

**Issue by NCTULouis** [chewing/chewing-editor#103]
You can add a tonal as a phase like this : 
 
![qwe](https://cloud.githubusercontent.com/assets/17592050/13903199/5eeec9e8-eeac-11e5-919d-dab504920cdb.png)

and the result is like this : 

![vvv](https://cloud.githubusercontent.com/assets/17592050/13903201/6ac859be-eeac-11e5-9c3d-6f6951b4da0e.png)

**Comment by kidwm** [comment]
@czchen maybe we should handle this case in libchewing?

**Comment by czchen** [comment]
@kidwm Current libchewing does not have word to bopomofo mapping table, so it cannot to the check.

Since there are many error handle cases that needs this table, we might need to add it to libchewing. @jserv, any suggestion?

**Comment by jserv** [comment]
@czchen : I agree with you. It would be nice if libchewing provides lookup function for Bopomofo mapping.
